libs/CCFbo/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 project( CC_FBO_LIB )
 
-add_library( ${PROJECT_NAME} STATIC )
+add_library( ${PROJECT_NAME} SHARED )
 
 add_subdirectory( include )
 add_subdirectory( src )
 
+target_compile_definitions( ${PROJECT_NAME} PRIVATE CCFBO_LIBRARY_BUILD )
+
 target_link_libraries( ${PROJECT_NAME} Qt5::Core Qt5::OpenGL Qt5::OpenGLExtensions )
```

Comment:
Shouldn't you call InstallSharedLibrary as well?

Reply:
Absolutely I should!

Thanks.